src/soorgeon/export.py

```diff
@@ -233,6 +234,28 @@ def _check(self):
         _check_functions_do_not_use_global_variables(code)
         _check_no_star_imports(code)
 
+    def _check_for_existing_output(self):
+        """
+        Check before refactoring. If the notebook itself
+        is supposed to generate some files, print a warning
+        telling the user to register them in the output section.
+        """
+        code = self._get_code()
+        tree = parso.parse(code)
+        output_lines = []
+        for i in tree.children:
```

Comment:
Are we iterating only the task folder here or the root project?

Reply:
what do you mean by task folder? Let's say we have a notebook.ipynb, it will iterate all the lines in the notebook. I don't think it has anything to do with file structures?

Reply:
I think Ido means checking the output tasks (since `soorgeon refactor` will break a single notebook into multiple files). This is execute before we do the refactoring process so I think it's fine, so we only have to check the input notebook as opposed to the output (multiple) notebooks/scripts

Reply:
Exactly what I meant, since it's pre refactoring - it includes all of the outputs.